test/functional/api/cas/cache_config.py

```diff
@@ -248,10 +243,7 @@ def __init__(self, threshold=None, trigger=None):
         self.trigger = trigger
```

Comment:
add types to__init__() functions in classes like PromotionParametersNhit, FlushParametersAlru, SeqCutoffParameters (check for others)